.github/workflows/test_template.yml

```diff
@@ -30,6 +30,11 @@ on:
         type: string
         required: false
         default: '["pip", ]'
+      # timeout:
```

Comment:
Do you mean for this to be commented out?

Reply:
I wanted to make it a parameter of this template it but does not seem to work. This is a temporary comment until I understand what is going on